src/types/notification.types.ts

```diff
@@ -6,3 +6,10 @@ export interface Notification {
   key: string;
   dismissed?: boolean;
 }
+
+export interface AddNotification {
+  message: string;
+  variant: VariantType;
+  key?: string;
+  dismissed?: boolean;
```

Comment:
brukes key og dismissed noengang for AddNotification? Tolker det som at AddNotification er den nye som vi genererer, så blir det automatisk opprettet en ny av typen Notification som har key og dismissed

Reply:
nei, det er sant! fjerner de propene 👍